docs/custom_promise_types/cfengine.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/false
```

Comment:
Commit title (first line), and PR title, should not end in punctuation (period).

https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#commit-message-example

Reply:
fixed!